contracts/src/BeefyClient.sol

```diff
@@ -243,8 +269,16 @@ contract BeefyClient {
             revert NotEnoughClaims();
         }
 
-        tickets[createTicketID(msg.sender, commitmentHash)] =
-            Ticket(msg.sender, uint64(block.number), uint32(vset.length), 0, keccak256(abi.encodePacked(bitfield)));
+        tickets[createTicketID(msg.sender, commitmentHash)] = Ticket({
+            account: msg.sender,
+            blockNumber: uint64(block.number),
+            validatorSetLen: uint32(vset.length),
+            numRequiredSignatures: computeNumRequiredSignatures(
+                vset.length, numRequiredSignatures, minNumRequiredSignatures
```

Comment:
```suggestion
                vset.length, signatureUsageCount, minNumRequiredSignatures
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/3da2bef5c6b3b13b02a5f4cd45a63e375c8c39e9